src/Monolog/Formatter/LineFormatter.php

```diff
@@ -258,7 +270,13 @@ private function formatException(\Throwable $e): string
                 }
             }
         }
-        $str .= '): ' . $e->getMessage() . ' at ' . $e->getFile() . ':' . $e->getLine() . ')';
+
+        $file = $e->getFile();
+        if ($this->basePath !== '') {
+            $file = str_replace($this->basePath, '', $e->getFile());
```

Comment:
Let's make sure we only replace the basePath at the beginning here

```suggestion
            $file = preg_replace('{^'.preg_quote($this->basePath).'}', '', $e->getFile());
```